src/bag/layout/template.py

```diff
@@ -2703,9 +2706,11 @@ def do_power_fill(self, layer_id: int, tr_manager: TrackManager,
         """
         # Value checks
         if sup_type.lower() not in ['vdd', 'vss', 'both']:
-            raise ValueError('sup_type has to be "VDD" or "VSS" or "both"(default)')
+            raise ValueError(
```

Comment:
You have a lot of these newlines happening. Our general style is to allow 100 characters in a line, so it's fine.

Reply:
I removed those linebreaks generated by the autoformatter. Thanks for the reminder.